src/styles/clock.scss

```diff
@@ -0,0 +1,83 @@
+.clock {
+  position: relative;
+  width: 600px;
+  height: 600px;
+
+  margin: 150px auto;
+
+  box-sizing: border-box;
+  border-radius: 50%;
+  border: 9px solid #424242;
+
+  &::after {
+    content: "";
+    position: absolute;
+    left: 0;
+    top: 0;
+    right: 0;
+    bottom: 0;
```

Comment:
just use width and height 100%?

Reply:
doesn`t work, there should be existing properties